ledger/src/blockstore_db.rs

```diff
@@ -1028,6 +1171,15 @@ fn get_cf_options(access_type: &AccessType) -> Options {
     options.set_level_zero_file_num_compaction_trigger(file_num_compaction_trigger as i32);
     options.set_max_bytes_for_level_base(total_size_base);
     options.set_target_file_size_base(file_size_base);
+
+    if C::NAME != columns::TransactionStatusIndex::NAME {
```

Comment:
maybe do this only if `AccessType::Primary`, exclude other modes including `Secondary`....